src/wechaty.ts

```diff
@@ -152,6 +161,10 @@ export class Wechaty extends EventEmitter implements Sayable {
    */
   private static globalInstance: Wechaty
 
+  private static globalPlugins: WechatyPlugin[] = []
```

Comment:
I'd like to suggest that we separate the effects between the Wechaty Instances, so that every Wechaty instances will set plugins by themselves, and only by themselves.

Reply:
explained below